tests/foreman/api/test_virt_who_config.py

```diff
@@ -0,0 +1,408 @@
+"""Test for virt-who configure API
+
+:Requirement: Virtwho-configure
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: notautomated
+
+:Upstream: No
+"""
+
+from robottelo.decorators import run_only_on, stubbed, tier1
+from robottelo.test import TestCase, APITestCase
+
+
+class VirtWhoConfigAPI(APITestCase):
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_vm_create(self):
+        """ Register a vm on virt-who hypervisor
+
+        :id: 701b22b8-fe92-4dd3-8d7d-c7b5efb7281b
+
+        :steps:
+            1. Create new VM in a supported hypervisor
+            2. Wait until the next virt-who report comes in.
+            3. Verify the VM is reported to satellite
+
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_vm_create(self):
+        """ Register a vm on non-virt-who hypervisor
+
+        :id: abecc851-2475-4455-9c12-63a73fcb09bb
+
+        :steps:
+            1. Verify there are no virt-who reports reported if there is no
+               change in guest-host mapping in hypervisor
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_config_update(self):
+        """Update a config, verify it it changed and redploy it
+
+        :id: 6b2cc2c3-959b-468b-9865-0f01decd2249
+
+        :steps:
+            1. Update a config, verify it it changed and redploy it
```

Comment:
typo `verify if it`

steps = update a config
expectedresults  = verify if it changed